src/Collection/ImmArray.php

```diff
@@ -484,4 +484,36 @@ protected function arraySort(callable $cb = null)
         }
         return static::fromArray($ar);
     }
+    
+    /**
+     * Checks if element exists in collection
+     *
+     * @param $element
+     * @param bool $strict use strict comparison?
+     * @return bool
+     */
+    public function has($element, $strict = false)
```

Comment:
Let's name this `includes`, to align it with the methods from the ecma `Array`.  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
done